sdk/tables/azure-data-tables/azure/data/tables/_models.py

```diff
@@ -617,8 +682,13 @@ def __init__(self, **kwargs):
         )
 
     def __str__(self):
+        # type: () -> str
         return self._str
 
+    def __repr__(self):
+        # type: () -> str
+        return u"AccountSasPermissions(_str={})".format(self._str)
```

Comment:
I wouldn't document `_str` - this isn't supposed to be public.
I would probably either document it as just the string:
`return "AccountSasPermissions({})".format(self._str)`
Or you could go fully verbose:
`return "AccountSasPermissions(read={}, write={}, ...)".format(self.read, self.write, ...)`

Reply:
Went with the latter for consistency